### PR TITLE
Build a ChatGPT-powered AI tool to cut the fluff from YouTube videos

### PR DESCRIPTION
**Pull request by viktoriasemaan**
# Content Pull Request Checklist

Please check to make sure your content submission meets the following requirements by adding an `x` between the brackets:

- [x] The title of this pull request reflects the title of my content.
- [x] I have reviewed the [Content Review Checklist](/content/blob/main/CONTENT_REVIEW_CHECKLIST.md).
- [x] This pull request contains one piece of content (post or page).

**Important: if you are submitting a PR to update existing content, please ensure you [sync'ed your fork](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/working-with-forks/syncing-a-fork) first before you created your branch.**

Is this pull request linked to a Content Proposal or Idea Suggestion Issue? If so, please link to it below using the number or full link.

```text
Resolves #5

or

Content for https://github.com/build-on-aws/content/issues/5
```

**By submitting this pull request, I confirm I own this content, have permission to use each image included in this content, and that it will be released under the [CC BY-SA 4.0 SA License](/LICENSE).**

### CODE REVIEW

File: tutorials/build-ai-with-hf-chatgpt/index.md [hunk 1]
```diff
@@ -0,0 +1,343 @@
...
+transcript = YouTubeTranscriptApi.get_transcript(video_id)
+
+# Concatenate the transcript into a single string
+transcript_text = ""
+for segment in transcript:
+    transcript_text += segment["text"] + " "
+print(transcript_text)
+```
+
+This code checks if a URL link is valid and then uses the *YouTubeTranscriptApi.get_transcript(video_id)* method to retrieve the YouTube transcript using the YouTube API. This method provides accurate and official captions associated with the video.
```
Comment (gaonkarr) [inline tutorials/build-ai-with-hf-chatgpt/index.md:172]: Is this https://pypi.org/project/youtube-transcript-api/ the YouTube API? Can we add a link to it?

File: tutorials/build-ai-with-hf-chatgpt/index.md [hunk 2]
```diff
@@ -0,0 +1,347 @@
...
+I will walk you through a step-by-step process on how to configure a free ML development environment using Amazon SageMaker Studio Lab, integrate it with open-source ML models from Hugging Face, and use OpenAI's ChatGPT APIs.
+
+You will be able to apply these steps to other use cases by selecting different ML models or adjusting ChatGPT prompts.
+
+| Attributes                |                                   |
+| ------------------- | -------------------------------------- |
+| ✅ AWS Level        | Intermediate - 200                         |
+| ⏱ Time to complete  | 30 minutes                             |
+| 💰 Cost to complete | Free when using the OpenAI API credit or less than $0.10      |
+| 🧩 Prerequisites    | - [Amazon SageMaker Studio Lab Account](https://studiolab.sagemaker.aws/?sc_channel=el&sc_campaign=datamlwave&sc_geo=mult&sc_country=mult&sc_outcome=acq&sc_content=build-ai-with-hf-chatgpt) <br>-  Foundational knowledge of Python    |                           |
```
Comment (gaonkarr) [inline tutorials/build-ai-with-hf-chatgpt/index.md:35]: If you have a link to a good basics of python, I would suggest added that

**Comment by gaonkarr** [comment]
@viktoriasemaan I have added couple of comments - link to a good python course, if you have and YouTube API. 

I have made minor changes, added tracking links and tags. Please do check these. You will have to fetch all the changes and then make your changes and push.

@damonk10 for visibility

**Review by gaonkarr — APPROVED** [review]
@viktoriasemaan I have added couple of comments - link to a good python course, if you have and YouTube API.

I have made minor changes, added tracking links and tags. Please do check these. You will have to fetch all the changes and then make your changes and push.

**Review by cherbk — APPROVED** [review]

**Pull request merged**